src/patternfly/components/Menu/menu.scss

```diff
@@ -648,6 +655,15 @@
   margin-right: var(--pf-c-menu__item-icon--MarginRight);
 }
 
+// Check
+.pf-c-menu__item-check {
+  margin-right: var(--pf-c-menu__item-check--MarginRight);
+
+  .pf-c-check {
+    grid-template-columns: auto; // update pf-c-check grid-template-columns to grid-auto-columns: 1fr at breaking change
```

Comment:
if you use the standalone modifier, you shouldn't need this

Reply:
I initially added `pf-m-standalone`, but that mod shifted the checkbox vertically. Are you seeing that?

Reply:
I thought one fewer modifier and this `grid-template-columns` update was better than adding a modifier, then adding a margin update to that modifier.

Reply:
Gotcha, tho this is one of the use cases the standalone modifier was created for. When we have a breaking change, it would be good if we could replace our use of regular checkboxes/radios (like in selectable tables) with the standalone check/radio component. I haven't looked at why the alignment is off - is it something we could fix by modifying the standalone variation or modifying the `__item-check` element you added here? If you still think it's better as is, we can leave it and revisit later since this component is beta.

Reply:
@mcoker updated!